packages/ui-extensions/src/surfaces/admin/components/CustomerSegmentTemplate/CustomerSegmentTemplate.ts

```diff
@@ -12,7 +12,7 @@ export interface CustomerSegmentTemplateProps {
   /* Localized description(s) of the template. */
   description: string | string[];
   /* DEPRECATED - Code snippet to render in the template with syntax highlighting. */
-  templateQuery: string;
+  templateQuery?: string;
```

Comment:
Also decided to make the deprecated field optional, so that I can remove support all at once in the segmentation-templates-app.